src/actix/mod.rs

```diff
@@ -50,7 +53,13 @@ pub fn init(toc: Arc<TableOfContent>, settings: Settings) -> std::io::Result<()>
                 )) // 32 Mb
                 .service(index)
                 .configure(config_collections_api)
-                .service(update_points)
+                .service(upsert_points)
+                .service(delete_points)
+                .service(set_payload)
+                .service(delete_payload)
+                .service(clear_payload)
+                .service(create_index)
+                .service(delete_index)
```

Comment:
Let's group all point update API in similar way as it is done in `config_collections_api`